rewrite-java/src/main/java/org/openrewrite/java/UpdateSourcePositions.java

```diff
@@ -135,7 +135,7 @@ public PrintOutputCapture<ExecutionContext> append(char c) {
             if (c == '\n') {
                 lineBoundary = true;
             }
-            return super.append(c);
+            return this;
```

Comment:
This looks like it would result in nothing being added to the buffer. Have you tested that this change works as intended?

Reply:
This subclass of `PrintOutputCapture` doesn't really care about proper printing. Content of the `out` `StringBuilder` is of no importance as far as I remember. The `PositionPrintOutputCapture` is only to track current offset, line and column of an AST node. I have sanity tested this stuff manually. Markers in the IDE show up and locations seem correct. However, I have just noticed that `UpdateSourcePositionsTest` is disabled. Perhaps, I'll see if can be enabled again.